index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives
+        <p>
+        <div id="img-div">
+            <img id="image" src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg"
+                alt="Dr. Norman Borlaug, third from the left">
+            <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+                increase
+                wheat yields - part of his life-long war on hunger.</div>
+        </div>
```

Comment:
- I think it would be better to use semantic tags
- Please use figure and figcaption tags instead of div tags

Reply:
> Please use figure and figcaption tags instead of div tags

- [ ] Hi, you are giving the code reviewer very good advice here. what would make your reviewer better is if you gave an explanation as to why you are making this suggestion. it's always good to explain the reasoning for your suggestion 👍

Reply:
## I think it would be better to use semantic tags

- [ ] Please use **figure** and **figcaption** tags instead of **div** tags, since it holds semantic value (it accurately describe the purpose of the element and the type of content that is inside them)